nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/SNMPGetter.java

```diff
@@ -84,17 +83,21 @@ public ResponseEvent get() {
      * @return the list of {@link TreeEvent}
      */
     public List<TreeEvent> walk() {
-        TreeUtils treeUtils = new TreeUtils(this.snmp, new DefaultPDUFactory());
-        @SuppressWarnings("unchecked")
-        List<TreeEvent> events = treeUtils.getSubtree(this.target, this.oid);
-        return events;
+        DefaultPDUFactory pduFactory = new DefaultPDUFactory();
+
+        // BEGIN WORKAROUND - For http://oosnmp.net/pipermail/snmp4j/2016-October/005749.html
```

Comment:
Great comment! For maintainability, do you think it would be helpful to put a comment in the pom to revisit this code if the snmp4j version is updated?